stories/Atom/Global/Logo/logo.stories.mdx

```diff
@@ -0,0 +1,91 @@
+import { Meta, Story, ArgsTable } from '@storybook/addon-docs';
+import { Logo } from './logo';
+import { Logowhite } from './logo-white';
+import { PnudLogo } from './pnud-logo';
+import { PnudLogowhite } from './pnud-logo-white';
+
+<Meta title="Atoms/Logos"  parameters={{
+    backgrounds: {
+      default: 'white',
+      values: [
+        { name: 'white', value: '#fff' },
+        { name: 'gray', value: '#CCC' },
+      ],
+    },
+  }}/>
+
+<style>{`
+  .subheading {
+    font-weight: 700;
+    font-size: 18px;
+    color: #666;
+    letter-spacing: 1.5px;
+    line-height: 24px;
+    margin-bottom: 12px;
+    margin-top: 40px;
+  }
+  div[id^="story--atoms-logos"] {
+    display: inline-block;
+    border: 1px solid #000;
+    padding: 10px;
+    align-items: center;
+    margin: 25px 25px;
+  }
+  .css-zgt7u1 {
+    background: #d8d8d8;
+  }
+`}</style>  
+
+# UNDP Logos
```

Comment:
Please revisit the documentation part for grammar error and also mention the correct usage of the inverted logo - it is used only for content in language - French, Spanish, and Portuguese.

Reply:
grammatical error and correct usage of the inverted logo is done, please check and merge.